isso/js/app/isso.js

```diff
@@ -71,6 +71,9 @@ var Postbox = function(parent) {
     var email_edit = function() {
         if (config["reply-notifications"] && $("[name='email']", el).value.length > 0) {
             $(".isso-notification-section", el).show();
+            if (config["reply-notifications-default-enabled"]) {
```

Comment:
This should be implemented in `isso/js/app/templates/postbox.js`

Something like:

```diff
diff --git a/isso/js/app/templates/postbox.js b/isso/js/app/templates/postbox.js
index a59d6a1..3f0a94b 100644
--- a/isso/js/app/templates/postbox.js
+++ b/isso/js/app/templates/postbox.js
@@ -1,8 +1,10 @@
 var html = function (globals) {
   var i18n = globals.i18n;
+  var conf = globals.conf;
   var author = globals.author;
   var email = globals.email;
   var website = globals.website;
+  var notify = conf['reply-notifications-default-enabled'] ? ' checked' : '';
 
   return "" +
 "<div class='isso-postbox'>"
@@ -39,7 +41,7 @@ var html = function (globals) {
   + "</section>"
   + "<section class='isso-notification-section'>"
     + "<label>"
-      + "<input type='checkbox' name='notification' />" + i18n('postbox-notification')
+      + "<input type='checkbox" + notify + " name='notification' />" + i18n('postbox-notification')
     + "</label>"
   + "</section>"
 + "</div>"
```